include/ginkgo/core/matrix/csr.hpp

```diff
@@ -140,8 +185,14 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         int64_t clac_size(const int64_t nnz) { return 0; }
     };
 
+    /**
+     * merge_path is a strategy_type which uses the merge_path algorithm.
```

Comment:
It would be nice if you can cite the paper(Duane Merill's paper I think) that first proposed the `merge_path` algorithm.